{
  "id": "EUVD-2025-5930",
  "description": "During an address list folding when a separating comma ends up on a folded line and that line is to be unicode-encoded then the separator itself is also unicode-encoded. Expected behavior is that the separating comma remains a plan comma. This can result in the address header being misinterpreted by some mail servers.",
  "datePublished": "Feb 28, 2025, 6:59:31 PM",
  "dateUpdated": "Feb 28, 2025, 8:32:56 PM",
  "baseScore": 2.3,
  "baseScoreVersion": "4.0",
  "baseScoreVector": "CVSS:4.0/AV:N/AC:H/AT:P/PR:L/UI:N/VC:L/VI:N/VA:N/SC:N/SI:N/SA:N",
  "references": [
    "https://nvd.nist.gov/vuln/detail/CVE-2025-1795",
    "https://github.com/python/cpython/issues/100884",
    "https://github.com/python/cpython/pull/100885",
    "https://github.com/python/cpython/pull/119099",
    "https://github.com/python/cpython/commit/09fab93c3d857496c0bd162797fab816c311ee48",
    "https://github.com/python/cpython/commit/70754d21c288535e86070ca7a6e90dcb670b8593",
    "https://github.com/python/cpython/commit/9148b77e0af91cdacaa7fe3dfac09635c3fe9a74",
    "https://mail.python.org/archives/list/security-announce@python.org/thread/MB62IZMEC3UM6SGHP5LET5JX2Y7H4ZUR",
    "https://mail.python.org/archives/list/security-announce@python.org/thread/MB62IZMEC3UM6SGHP5LET5JX2Y7H4ZUR/"
  ],
  "aliases": [
    "CVE-2025-1795",
    "GHSA-c266-vjjr-2v8j"
  ],
  "assigner": "PSF",
  "epss": 0.17,
  "enisaIdProduct": [
    {
      "id": "1d4895bb-3c3d-388c-bcc0-38d4a45a9e8f",
      "product": {
        "name": "CPython"
      },
      "product_version": "3.13.0a1 <3.13.0a5"
    },
    {
      "id": "7fa27a10-9465-3fdb-99bc-187b92289b12",
      "product": {
        "name": "CPython"
      },
      "product_version": "0 <3.11.9"
    },
    {
      "id": "c2a2104e-1a9c-34a5-8e85-f65cf7cf5fc3",
      "product": {
        "name": "CPython"
      },
      "product_version": "3.12.0 <3.12.3"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "41075d37-2e49-3c98-b1bc-edef239f68d6",
      "vendor": {
        "name": "Python Software Foundation"
      }
    }
  ],
  "cve": "CVE-2025-1795"
}
